docs/src/Color.doc.js

```diff
@@ -0,0 +1,61 @@
+// @flow strict
+import type { Node } from 'react';
+import { Flex } from 'gestalt';
+import MainSection from './components/MainSection.js';
+import PageHeader from './components/PageHeader.js';
+import ColorPalette from './components/ColorPalette.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Color"
+    description={`
+    Our color palettes are shared between Brand and Gestalt, and represent our full range of options. For further Brand guidance, please reference the [Brand guidelines website](https://brand.pinterest.com/).
+    `}
+  />,
+);
+
+card(
+  <MainSection
+    name="Full Palettes"
+    description="In order to ensure accessible contrast for color pairings, we require a dark font color to be used for any colors 400 and below. For 500 and above, we recommend using white."
+  >
+    <MainSection.Subsection title="Colors">
+      <Flex direction="column" gap={12}>
```

Comment:
nit: rather than hardcode all this markup manually, a couple of helper components that you can use to map an array of `{ name, tokenId }` would be nice. This would be annoying to update if we want to, say, adjust the `gap`.

Reply:
I left this for now, but I can revisit when I do the design tokens page